TipCatDotNet.Api/Models/Payments/Enums/AllowedPaymentMethods.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.ComponentModel;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace TipCatDotNet.Api.Models.Payments.Enums
+{
+    public static class AllowedPaymentMethods
```

Comment:
The name looks like it is an entity, but not a service. Probably should be `PaymentMethodService.GetAllowed()` or `PaymentMethodService.Get()`